engines/sci/engine/kvideo.cpp

```diff
@@ -113,6 +136,12 @@ void playVideo(Video::VideoDecoder &videoDecoder) {
 	}
 }
 
+void playVideo(Video::VideoDecoder &videoDecoder) { // For the debugger
+	playVideo(videoDecoder, nullptr);
+}
+
+MovieConfig *_movieConf = nullptr;
```

Comment:
Can we hang this off something? I think we've eliminated all the floating state objects from SCI, so for consistency this should be a field in one of our many objects.

Reply:
Maybe I really revert those kVideo changes which really serve only documentary purposes. It would be different if the engine really made use of the x, y, playFrom or playTo parameters, but it doesn't. Seems like the devs were a bit undecided what to do. The first video (the logo) actually has a y = 5 vertical positioning, but due to several contradicting MCI_PUT calls it will just get centered, ignoring the coords.

Reply:
> Maybe I just revert those kVideo changes which really serve only documentary purposes. It would be different if the engine made use of the x, y, playFrom or playTo parameters, but it doesn't. Seems like the devs were a bit undecided what to do. The first video (the logo) actually has a y = 5 vertical positioning, but due to several contradicting MCI_PUT calls it will just get centered, ignoring the coords.

It would be better to disable this unused code, so that if we ever find a game variant that actually uses this code, it won't be wasted :)

Reply:
Okay, then I'll find a another way to get rid of the global var.